### PR TITLE
Update and add FSV nodes

### PR DESCRIPTION
**Pull request by dstrande**

### CODE REVIEW

File: pyproject.toml
Comment (itsjoeoui) [inline pyproject.toml:1]: typer is already in dev dependencies, should not add here again. 
steps to fix:
1. remove from dep
2. regen poetry.lock

**Review by itsjoeoui — DISMISSED** [review]

File: pyproject.toml
Comment (dstrande) [reply pyproject.toml:1]: Hmmm I didn't mean to do this, I'll fix it. Every time I go to WSL to sync I have to add it to poetry haha.

File: blocks/DSP/PEAK_DETECTION/PEAK_DETECTION.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    -------
+    OrderedPair
+        x: x axis location for peaks
+        y: peaks
+    """
+
+    if height == "":
+        height = None
+    else:
+        height = float(height)
```
Comment (39bytes) [inline blocks/DSP/PEAK_DETECTION/PEAK_DETECTION.py]: You can 1 line all of these
```python
height = float(height) if height != "" else None
```

File: blocks/ETL/TRANSFORM/TYPE_CASTING/ORDERED_PAIR_2_VECTOR/ORDERED_PAIR_2_VECTOR.py [hunk 2]
```diff
@@ -1,9 +1,15 @@
 from flojoy import flojoy, Vector, OrderedPair
+from typing import TypedDict
+
+
+class resultSplit(TypedDict):
```
Comment (39bytes) [inline blocks/ETL/TRANSFORM/TYPE_CASTING/ORDERED_PAIR_2_VECTOR/ORDERED_PAIR_2_VECTOR.py]:
```suggestion
class ResultSplit(TypedDict):
```

File: blocks/ETL/TRANSFORM/TYPE_CASTING/ORDERED_PAIR_2_VECTOR/ORDERED_PAIR_2_VECTOR.py [hunk 3]
```diff
@@ -12,8 +18,9 @@ def ORDERED_PAIR_2_VECTOR(default: OrderedPair) -> Vector:
...
     -------
-    Vector
-        The y component of the input OrderedPair.
+    TypedDict:
+        x: Vector from input x
+        y: Vector from input y
     """
 
-    return Vector(v=default.y)
+    return resultSplit(x=Vector(v=default.x), y=Vector(v=default.y))
```
Comment (39bytes) [inline blocks/ETL/TRANSFORM/TYPE_CASTING/ORDERED_PAIR_2_VECTOR/ORDERED_PAIR_2_VECTOR.py]:
```suggestion
    return ResultSplit(x=Vector(v=default.x), y=Vector(v=default.y))
```

File: blocks/ETL/TRANSFORM/TYPE_CASTING/ORDERED_PAIR_2_VECTOR/ORDERED_PAIR_2_VECTOR.py [hunk 4]
```diff
@@ -1,9 +1,15 @@
...
+
+class resultSplit(TypedDict):
+    x: Vector
+    y: Vector
 
 
 @flojoy
-def ORDERED_PAIR_2_VECTOR(default: OrderedPair) -> Vector:
-    """Convert an OrderedPair DataContainer to a Vector DataContainer.
+def ORDERED_PAIR_2_VECTOR(default: OrderedPair) -> resultSplit:
```
Comment (39bytes) [inline blocks/ETL/TRANSFORM/TYPE_CASTING/ORDERED_PAIR_2_VECTOR/ORDERED_PAIR_2_VECTOR.py]:
```suggestion
def ORDERED_PAIR_2_VECTOR(default: OrderedPair) -> ResultSplit:
```

**Review by 39bytes — APPROVED** [review]

**Pull request merged**